.gitignore

```diff
@@ -5,3 +5,4 @@
 # Temporary Build Files
 build/_output
 build/_test
+bin/*
```

Comment:
This one is also from the example.

https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/v4/memcached-operator/.dockerignore#L3